thanks.html

```diff
@@ -530,8 +530,9 @@ <h5 class="taf-heading">Donate</h5>
 		method: 'GET',
 		url: shareURL,
 	}).done(function(shareURL) {
-		$('#page-link-url-general').val(shareURL);
 		navigator.clipboard.writeText(shareURL);
+	}).fail(function() {
+		navigator.clipboard.writeText(window.location.href);

```

Comment:
From the [ticket convo](https://github.com/350org/action-tools/issues/49#issuecomment-1625588603) it seems that handling the edge case of the thank you page being visited directly (therefore no valid action_id), is not needed? Just want to confirm this. It seems if we wanted to handle that, the page name could be added to the original share url as a fallback when action_id is missing.

Reply:
nope, I think that's not a case we want to handle - I had the page name in there before but changed it to only include `action_id` based on Rachel's feedback